xbmc/cores/VideoPlayer/DVDInputStreams/DVDFactoryInputStream.cpp

```diff
@@ -57,6 +57,9 @@ std::shared_ptr<CDVDInputStream> CDVDFactoryInputStream::CreateInputStream(IVide
       return std::shared_ptr<CInputStreamAddon>(new CInputStreamAddon(addonInfo, pPlayer, fileitem));
   }
 
+  if (fileitem.GetProperty("inputstreamclass").asString() == "inputstream.ffmpeg")
```

Comment:
why did you create a define and don't use it here?

Reply:
I’ll take that one on the chin 😉